--- .clang-format	2020-07-14 15:27:28.000000000 +0900
+++ .clang-format-6.0	2020-07-14 14:39:54.000000000 +0900
@@ -28,6 +28,7 @@
     AfterObjCDeclaration: false
     AfterStruct: false
     AfterUnion: false
+    AfterExternBlock: false # Unknown to clang-format-5.0
     BeforeCatch: false
     BeforeElse: false
     IndentBraces: false
@@ -52,11 +53,13 @@
 DisableFormat: false
 ExperimentalAutoDetectBinPacking: false
 FixNamespaceComments: false # Unknown to clang-format-4.0
+IncludeBlocks: Preserve # Unknown to clang-format-5.0
 IncludeCategories:
   - Regex: '.*'
     Priority: 1
 IncludeIsMainRegex: '(Test)?$'
 IndentCaseLabels: false
+IndentPPDirectives: None # Unknown to clang-format-5.0
 IndentWidth: 8
 IndentWrappedFunctionNames: false
 JavaScriptQuotes: Leave
@@ -66,6 +69,7 @@
 MacroBlockEnd: ''
 MaxEmptyLinesToKeep: 1
 NamespaceIndentation: Inner
+ObjCBinPackProtocolList: Auto # Unknown to clang-format-5.0
 ObjCBlockIndentWidth: 8
 ObjCSpaceAfterProperty: true
 ObjCSpaceBeforeProtocolList: true
@@ -86,7 +90,10 @@
 SpaceAfterCStyleCast: false
 SpaceAfterTemplateKeyword: true
 SpaceBeforeAssignmentOperators: true
+SpaceBeforeCtorInitializerColon: true # Unknown to clang-format-5.0
+SpaceBeforeInheritanceColon: true # Unknown to clang-format-5.0
 SpaceBeforeParens: ControlStatements
+SpaceBeforeRangeBasedForLoopColon: true # Unknown to clang-format-5.0
 SpaceInEmptyParentheses: false
 SpacesBeforeTrailingComments: 1
 SpacesInAngles: false
