--- .clang-format-6	2020-07-18 02:43:10.000000000 +0900
+++ .clang-format-9	2020-07-18 02:43:14.000000000 +0900
@@ -4,6 +4,7 @@
 AlignAfterOpenBracket: Align
 AlignConsecutiveAssignments: false
 AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true # Unknown to clang-format-8.0
 AlignEscapedNewlines: Left # Unknown to clang-format-4.0
 AlignOperands: true
 AlignTrailingComments: false
